docs/DeveloperGuide.md

```diff
@@ -310,22 +330,24 @@ Our product aims to optimize managing of inventory and cash flow in a restaurant
+| `* * *`                       | cafe manager is responsible for managing pantry stock                 | track the inventory levels for ingredients and supplies | know what ingredients I need to restock                                                 |
+| `* * *`                       | cafe manager is responsible for managing pantry stock                 | buy ingredients                                         | restock low stock ingredients                                                           |
+| `* * *`                       | cafe owner who is also the chef                                       | view the ingredients needed for a dish                  | know what ingredients to use when cooking a dish                                        |
+| `* * *`                       | cafe owner who wants to maximise profit                               | edit the price of the dish                              | increase the price of the dish when there is inflation                                  |
+| `* * *`                       | cafe owner who cares about the sales of the cafe                      | view the sales of the cafe                              | know whether my cafe is profiting                                                       |
+| `* * *`                       | cafe owner who works 7 days a week                                    | save the menu, pantry stock and order                   | have access to the same menu, pantry stock and orders when I go back to work            |
+| `* * *`                       | cafe owner who is responsible for placing order                       | add order                                               | ask the chef to cook the order                                                          |
+| `* *`                         | cafe manager who is responsible for drafting the menu                 | view the menu                                           | keep track of what dish we have                                                         |
+| `* *`                         | cafe owner who working 7 days a week                                  | fast forward to the next day                            | close the cafe and call it a day when I am tired                                        |
+| `* *`                         | clumsy cafe owner who works 7 days a week                             | go back to the previous day                             | still accept order from the previous day if I accidentally fast forward to the next day | 
+| `* *` <br>(to be implemented) | cafe owner who is interested to know the popularity of the menu items | view the rank of popularity based on order history      | adjust the pricing or remove the dish that is not popular                               |
 
 *{More to be added}*
```

Comment:
is there any more to be added? if not perhaps can delete this line

Reply:
I will delete this line in my latest PR!